views/play.erb

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Hi, <%= @player %>! | Rock, Paper, Scissors</title>
+  <style>
+    body {
+      display: flex;
```

Comment:
It's great that you've added some style to make it look better.  I'm just curious why you've put the css in the same file as the HTML.  I thought it's best practice to put the css in a separate file, then import it.  I guess it's personal preference though.